invoices/invoiceregistry.go

```diff
@@ -491,6 +496,18 @@ func (i *InvoiceRegistry) NotifyExitHopHtlc(rHash lntypes.Hash,
 			return nil, errNoUpdate
 		}
 
+		// If expiry is not set, then assume the default 3600 seconds.
+		invExpiry := inv.Expiry
+		if inv.Expiry == 0 {
+			invExpiry = defaultInvoiceExpiry
+		}
+
+		// Check invoice expiry.
```

Comment:
Comments in this diff are borderline compliant with the comments section: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#43-code-documentation-and-commenting 

> Comments in the body of the code are highly encouraged, but they should explain the intention of the code as opposed to just calling out the obvious

Reply:
PTAL